src/Sentry.Unity.Editor.iOS/BuildPostProcess.cs

```diff
@@ -137,19 +136,6 @@ internal static void SetupSentry(SentryUnityOptions options,
                 sentryXcodeProject.AddNativeOptions(options, NativeOptions.CreateFile);
                 sentryXcodeProject.AddSentryToMain(options);
             }
-
-            if (cliOptions != null && cliOptions.IsValid(logger, EditorUserBuildSettings.development))
-            {
-                logger.LogInfo("Automatic symbol upload enabled. Adding script to build phase.");
-
-                SentryCli.CreateSentryProperties(pathToProject, cliOptions, options);
-                SentryCli.SetupSentryCli(pathToProject, RuntimePlatform.OSXEditor);
-                sentryXcodeProject.AddBuildPhaseSymbolUpload(cliOptions);
-            }
-            else if (options.Il2CppLineNumberSupportEnabled)
-            {
-                logger.LogWarning("The IL2CPP line number support requires the debug symbol upload to be enabled.");
-            }
```

Comment:
I moved this out of `SetupSentry` into `SetupSentryCli`.
`SetupSentry` is responsible for adding the framework and modifying the Xcode project to have the options and bridge set.
`SetupSentryCli` adds the executable, writes the `.properties` and adds the script to invoke the symbol upload.